saleor/graphql/giftcard/tests/mutations/test_gift_card_bulk_deactivate.py

```diff
@@ -0,0 +1,134 @@
+import graphene
+
+from .....giftcard import GiftCardEvents
+from .....giftcard.models import GiftCard, GiftCardEvent
+from ....tests.utils import assert_no_permission, get_graphql_content
+
+MUTATION_GIFT_CARD_BULK_DEACTIVATE = """
+    mutation GiftCardBulkDeactivate($ids: [ID]!) {
+        giftCardBulkDeactivate(ids: $ids) {
+            count
+            errors {
+                code
+                field
+            }
+        }
+    }
+"""
+
+
+def test_gift_card_bulk_deactivate_by_staff(
+    staff_api_client,
+    gift_card,
+    gift_card_expiry_period,
+    gift_card_expiry_date,
+    permission_manage_gift_card,
+):
+    # given
+    gift_cards = [gift_card, gift_card_expiry_period, gift_card_expiry_date]
+
+    ids = [graphene.Node.to_global_id("GiftCard", card.pk) for card in gift_cards]
+    variables = {"ids": ids}
+
+    # when
+    response = staff_api_client.post_graphql(
+        MUTATION_GIFT_CARD_BULK_DEACTIVATE,
+        variables,
+        permissions=(permission_manage_gift_card,),
+    )
+
+    # then
+    content = get_graphql_content(response)
+    data = content["data"]["giftCardBulkDeactivate"]
+
+    assert data["count"] == len(ids)
+    events = GiftCardEvent.objects.all()
+    assert events.count() == 3
+    assert {event.gift_card_id for event in events} == {
+        gift_card.id,
+        gift_card_expiry_period.id,
+        gift_card_expiry_date.id,
+    }
+    assert {event.type for event in events} == {GiftCardEvents.DEACTIVATED}
+
+
+def test_gift_card_bulk_deactivate_by_app(
```

Comment:
This test should also check that these gift cards are deactivated after this mutation.